vsintegration/src/FSharp.UIResources/Strings.resx

```diff
@@ -231,6 +231,21 @@
   <data name="Find_References_Performance" xml:space="preserve">
     <value>Find References Performance Options</value>
   </data>
+  <data name="Language_Service_Settings" xml:space="preserve">
```

Comment:
It would be nice to add them [here](https://github.com/T-Gro/fsharp/blob/main/vsintegration/src/FSharp.Editor/FSharp.Editor.resx) as well to have them as searchable tokens within VS options.

Reply:
Those are temporary, no ned to promote them